app/assets/stylesheets/sessions.scss

```diff
@@ -1,3 +1,30 @@
-// Place all the styles related to the sessions controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+body {
+    font-family: 'Lato', sans-serif;
```

Comment:
please put things related to body into application.scss. Also, #login should not be nested inside body. 

Please change this behaviour for all your scss